### DISCUSSION TITLE
Is strategy an Option for computePosition?

### DISCUSSION THREAD

**Discussion by ollicle** [floating-ui/floating-ui#2775] [Q&A] [answered]
Using the docs page search for “strategy” – “strategy computePosition” appears in the results.
Although, when I click that search result the item is missing from the documentation. It only appears as a bullet point under **On this page**.

I’m hoping it is possible to benefit from the flip middleware to calculate the position when using an absolutely positioned floating element (rather than fixed)?

Setting a fixed position on my element results in the correct position. However, fixed positioning is awkward when the floating element loses it's position relative to the reference element when scrolling the page.

In case it is relevant my floating element is using popover to place it in the top layer.

**Comment by atomiks** [comment, marked as answer]
Yes, it is. You must have selected "Core" from the red dropdown at the top right? It appears for DOM, React, React DOM, and Vue, but not React Native or Core (the latter is a bug, since it exists for Core). But you probably should be selecting "DOM"?

**Reply by ollicle** [reply]
Thanks @atomiks that’s exactly what I was doing. DOM is what I want 🙏
